### PR TITLE
[node] Add softmax kernel.

### PR DESCRIPTION
**Pull request by annxingyuan**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/2767)

### CODE REVIEW

**Comment by annxingyuan** [comment]
@lina128 yes - thanks for the suggestion :) would you mind taking another look?
